src/netif.c

```diff
@@ -2069,6 +2088,20 @@ static void lcore_job_recv_fwd(void *arg)
                 eth_hdr = rte_pktmbuf_mtod(mbuf, struct ether_hdr *);
                 /* reuse mbuf.packet_type, it was RTE_PTYPE_XXX */
                 mbuf->packet_type = eth_type_parse(eth_hdr, dev);
+
+                /*
+                   * In NETIF_PORT_FLAG_FORWARD2KNI mode.
```

Comment:
1. comment indent
use
```
/*
 *
 */
```
instead of
```
/*
    *
    *
    */
```

2. be sure each line not exceed  80 chars.
 Yes, a lot of codes in `DPVS`, like netif.c donot follow this rule, but for new added codes, I think we should follow that.